site/homepage/content/release-notes.md

```diff
@@ -22,6 +22,13 @@ title = "Release Notes"
 * The Hono chart can now also be deployed using the recently released Helm 3. In fact,
   Helm 3 is now the recommended way for deploying Hono as it doesn't require installation
   of any Helm specific components to the Kubernetes cluster.
+* A potential issue has been resolved where some command messages might not have been sent
+  to the corresponding gateway. The scenario here involves the gateway sending event/telemetry
+  messages via HTTP with a `hono-ttd` header in order to receive commands, and doing so
+  with multiple concurrent requests for *different* devices. The solution makes it
+  necessary that gateways can no longer send a event/telemetry message via HTTP with a 
+  `hono-ttd` header in order to receive commands for *any* device that has last sent a
+  telemetry or event message via this gateway.
```

Comment:
IMHO this constitutes a breaking API change, in this case the HTTP adapter's public API. I therefore do not think that we can include such a change in a service release. Another option would be to make this behavior change configurable per tenant with a default of the *existing, faulty* behavior.

Reply:
Right. I think introducing an option in the tenant data looks doable here. I'll explore this and update this PR.

Reply:
Ok, so we would introduce a new boolean flag:
- value `false` (default) means the behaviour stays the same as today: 
  - a gateway can still send *one* HTTP request to request a command from *any* device that is connected to the gateway
  - concurrent `ttd` requests from the gateway for different devices have the described issue that commands could get lost
- value `true` means
  - the concurrency issue doesn't exist (gateway-mapping is disabled)
  - but the gateway cannot send *one* HTTP request to request a command from *any* device

I would name the flag `support-concurrent-gateway-device-command-requests` (or `support-concurrent-gateway-device-command-subscriptions` or something like that).

The question is now where to put it in the tenant data.
I think we could put it in the `adapters` field of the tenant data.
If we define the option in the `Adapter` schema type, we would either need to specify in the documentation that this only applies to the HTTP adapter or we implement it for the other adapters as well (although for them it wasn't seen as that necessary - see #1644). (Alternative would be putting the option in the `adapters/ext` section, although this option is not really external here so I think it wouldn't fit.)
Or we use this as motivation to define a new generic `adapters/flags` (or something like that) container field, under which any kind of adapter-type specific flags shall go.
I would tend towards the last option. WDYT?

Reply:
> Alternative would be putting the option in the adapters/ext section, although this option is not really external here so I think it wouldn't fit.

`ext` here stands for *extensions* ... FMPOV it the new property would be a good fit for this category as it only applies to the HTTP adapter and we intend to remove the flag anyway once we have moved on to single command receiver per protocol adapter instance.

We could simply document this extension property in the HTTP adapter admin guide.

Reply:
(I thought `ext` was reserved for extensions outside of the Hono API, ie. external to it. But I guess it's not that strict.)
Ok, yes, that's also the least invasive approach.